src/component/Registration/Signup.css

```diff
@@ -0,0 +1,85 @@
+.registrationForm {
+    background-color: #fff;
+    color: black;
+    width: 400px;
+    border-radius: 8px;
+    padding: 8px 8px; 
+  }
+
+  .registrationFolder{
+      height: 100vh;
+      width: 100%;
+      background-color: aliceblue;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+  }
+
+  .registrationBody{
+      width: 90%;
+      max-width: 500px;
+      margin: 0 auto;
+      padding: 20px;
+      box-shadow: 0px 0px 20px #00000010;
+      border-radius: 8px;
+      margin-bottom: 20px;
+  }
+
+  .formGroup {
+      width: 100%;
+      margin-top: 20px;
+      font-size: 20px;
+  }
+
+  .formGroup input,
+  .formGroup textarea {
+      width: 100%;
+      padding: 5px;
+      font-size: 18px;
+      border: 1px solid #5ea51d;
+  }
+
+  .submitBtn{
+      width: 74%;
+      border: none;
+      outline: none;
+      padding: 20px;
+      font-size: 24px;
+      border-radius: 8px;
+      margin-top: 2rem;
+      background-color: #5ea51d;
+  }
+
+.agent{
+  color: black;
+  font-size: 14px;
+}
+
+.agentSignin {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 1rem;
+}
+
+.submitBtn {
+  font-size: 20px;
+}
+
+@media (max-width: 769px) and (min-width: 360px) {
+
+    .nav-main {
+        width:100%;
+        margin:0;
+    }
+   
+    .nav-icons {
```

Comment:
Can you stick with a casing pattern? I see you have both camel case (camelCase) and kebab case (kebab-case), choosing one makes your code easier to read and shows some sense of originality.

Reply:
resolved also